### PR TITLE
Added basic styling for pagination

### PR DESCRIPTION
**Pull request by Roene-JustBetter**
Added basic styling for pagination on category pages. Importants needed because of the reactive search components.

### CODE REVIEW

**Comment by Roene-JustBetter** [comment]
@jordythevulder @JimmyHoenderdaal

File: resources/css/components/pagination.css [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+.pagination-button:not(:first-child):not(:last-child) {
+    @apply !m-0 !size-14 max-md:flex-1 max-md:w-auto;
+}
+
+.pagination-button:first-child,
+.pagination-button:last-child {
+    @apply h-14 px-6 max-md:!m-0;
+}
+
+.pagination-button[disabled] {
```
Comment (JimmyHoenderdaal) [inline resources/css/components/pagination.css:30]: you can do this instead on the .pagination-button
`@apply disabled:opacity-40`
maybe also add a cursor-not-allowed it too

Comment (Roene-JustBetter) [reply resources/css/components/pagination.css:30]: The button gets a disabled attribute so the cursor-not-allowed is default here.

**Review by JimmyHoenderdaal — APPROVED** [review]

**Comment by JimmyHoenderdaal** [comment]
Looks good to me 👍

**Comment by royduin** [comment]
Old vs new visually?

**Comment by Roene-JustBetter** [comment]
> Old vs new visually?

Old:
Scherm­afbeelding 2024-06-18 om 15 27 36

New:
Scherm­afbeelding 2024-06-18 om 15 40 22

Also needed to update Tailwind. In Tailwind 3.4 you now can use `size-14` instead of `w-14 h-14`

**Comment by royduin** [comment]
So what's the status on this one? Everybody approves? @jordythevulder @JimmyHoenderdaal @Roene-JustBetter?

**Review by jordythevulder — APPROVED** [review]

**Comment by royduin** [comment]
Why are the tests failing? And the frontend dependency updates are required for this? If not; better to have a separated PR with all dependencies updated.

**Comment by Roene-JustBetter** [comment]
> Why are the tests failing? And the frontend dependency updates are required for this? If not; better to have a separated PR with all dependencies updated.

I checked and it is not needed. I will close this Pull Request and make a new Pull Request with only the changes that are needed.

**Pull request closed without merging**